TimberAPI/CustomObjectSystem/CustomObjectRegistry.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace TimberbornAPI.CustomObjectSystem
+{
+    public class CustomObjectRegistry : ICustomObjectRegistry
+    {
+		private List<GameObject> CustomObjects = new();
+        private Boolean Accessed = false;
+
+		public void AddGameObject(GameObject gameObject)
```

Comment:
What happend with the styling in this class 0.o

Reply:
It’s not like that on my local o,o